scripts/tests/localsetup_tests.py

```diff
@@ -80,10 +80,8 @@ def test_opbeans_dotnet(self):
                           max-size: '2m'
                           max-file: '5'
                     depends_on:
-                      elasticsearch:
-                        condition: service_healthy
-                      apm-server:
-                        condition: service_healthy
+                      - apm-server
+                      - elasticsearch
```

Comment:
Missing the conditions, I guess this is the reason of the current test failures in the CI